packages/react-devtools-scheduling-profiler/src/EventTooltip.css

```diff
@@ -0,0 +1,79 @@
+.Tooltip {
+  position: fixed;
+  display: inline-block;
+  border-radius: 0.125rem;
+  max-width: 300px;
+  padding: 0.25rem;
+  user-select: none;
+  pointer-events: none;
+  background-color: #ffffff;
+  border: 1px solid #ccc;
+  box-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
+  font-size: 11px;
+}
+
+.Divider {
+  height: 1px;
+  background-color: #aaa;
+  margin: 0.5rem 0;
+}
+
+.DetailsGrid {
+  display: grid;
+  padding-top: 5px;
+  grid-gap: 2px 5px;
+  grid-template-columns: min-content auto;
+}
+
+.DetailsGridLabel {
+  color: #666;
+  text-align: right;
+}
+
+.DetailsGridURL {
+  word-break: break-all;
+  max-height: 50vh;
+  overflow: hidden;
+}
+
+.FlamechartStackFrameName {
+  word-break: break-word;
+  margin-left: 0.4rem;
+}
+
+.ComponentName {
+  font-weight: bold;
+  word-break: break-word;
+  margin-right: 0.4rem;
+}
+
+.ComponentStack {
+  overflow: hidden;
+  max-width: 35em;
+  max-height: 10em;
+  margin: 0;
+  font-size: 0.9em;
+  line-height: 1.5;
+  -webkit-mask-image: linear-gradient(
+    180deg,
+    #fff,
+    #fff var(--gradient-height),
+    transparent
+  );
+  mask-image: linear-gradient(
+    180deg,
+    #fff,
+    #fff var(--gradient-height),
+    transparent
+  );
+  white-space: pre;
+  --gradient-height: 5em;
```

Comment:
Oh, you used that to add a fade to the bottom half of taller component stacks:

|With this variable|If this line is deleted|
|-|-|
|![image](https://user-images.githubusercontent.com/12784593/90651206-5562b180-e26f-11ea-8ced-debd8857616a.png)|![image](https://user-images.githubusercontent.com/12784593/90651251-63183700-e26f-11ea-854e-4dc487f7b944.png)|

I think it's a variable because it's used in 2 places (i.e. both a prefixed and not prefixed version of `mask-image`). I'll be happy to inline it or remove the gradient entirely

Reply:
Yeah, I think since it's only used in two places (also in this file) we should just inline it. That's what my comment was trying to say, rather than asking what the CSS style itself was being used for :grin:

Reply:
Oops, my bad! 😅 I'll inline this